<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hen_dialog</class>
 <widget class="QDialog" name="Hen_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>652</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="upgrade_hen_butt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius:40px;
color: rgb(120, 60, 0);
background-color: rgb(51, 210, 218);</string>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="dane_l">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>250</y>
     <width>551</width>
     <height>451</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="collectegg_butt">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="danedadan_butt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>-10</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>capacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="hen_counter_l">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>520</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(51, 210, 218);

color:rgb(255, 255, 255);
border-radius:20px;
</string>
   </property>
   <property name="text">
    <string>  30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lvl</string>
   </property>
  </widget>
  <widget class="QLabel" name="capacity_hen_l">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lvl_hen_l">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
